.github/workflows/ci.yml

```diff
@@ -60,7 +60,6 @@ jobs:
           elif [[ "$job" == "test" ]]; then
             ROOT=$(pwd)
             KUBEBUILDER_ASSETS=${ROOT}/output/bin/kubebuilder/bin make test
```

Comment:
`make test` exists here and below. Is this expected behavior?

Reply:
Yes, it is expected. 

I just removed the `CI=1 make coverage` which would upload coverage to codecov. And do not change others.

I am not sure we should run `make test` here, it seems duplicated, so I leave it alone without any changes. Maybe we could refine it in next PRs when necessary.

Reply:
Know about it. We can decide whether to remove the original test job later.